db/sql/sql.go

```diff
@@ -311,6 +321,10 @@ func (db *DB) Begin() (tx transaction.Transaction, err error) {
 		closed:         false,
 		isolationLevel: transaction.RepeatableRead,
 	}
+	if os.Getenv("FUZZY_DB_TX") == "true" {
```

Comment:
Fuzzy comes from term 'fuzzy testing' - here we randomly insert errors in DB calls. Do you suggest renaming this var to some other more meaningful name?